CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * In Settings > Users > Patron Blocks > Conditions, mark "Message to be displayed" as required. Refs UIU-2487.
 * Enable dependabot. Refs UIU-2747, FOLIO-3664.
 * Suppress delete of users stored in a configuration entry. Refs UIU-2738.
+* Remove deps related to deprecated BigTest infrastructure. Refs UIU-2745.
```

Comment:
Can we update the commit message to visualize that the tests have also been removed?